src/components/UserStatistics.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+import green from "@material-ui/core/colors/green";
+import red from "@material-ui/core/colors/red";
+import grey from "@material-ui/core/colors/grey";
+import moment from "moment";
+import { Pie } from "react-chartjs-2";
+
+import Loading from "./Loading";
+
+const useStyles = makeStyles((theme) => ({
+  statisticsPanel: {
+    background: theme.palette.background.default,
+    padding: theme.spacing(2),
+    borderRadius: 4,
+  },
+  stats: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+  },
+}));
+
+function UserStatistics({ gamesData, userId, loadingGames }) {
+  const classes = useStyles();
+
+  if (loadingGames || !gamesData) {
```

Comment:
Checking `loadingGames` is already handled in the outside container. This prop is unnecessary.